src/utility.ts

```diff
@@ -0,0 +1,85 @@
+
+import { Metadata } from "@grpc/grpc-js";
+import json_bigint from "json-bigint";
+import { TransactionCtx as ProtoTransactionCtx } from "./proto/server/v1/api_pb";
+import { Session } from "./session";
+import { Filter, Filters, LogicalFilter, LogicalFilters, ReadFields, UpdateFields } from "./types";
+
+export const Utility = {
```

Comment:
is there a way so the visibility of this class will not be exposed to consumer of this library, but just internal to the library? or is that how it is already?

Reply:
See my comment above, we will only export the items that we want to publish

Reply:
ok so we will still have an opportunity to select what we want to expose to users. sounds good in that case.